ui/app/AppLayouts/Profile/views/wallet/MainView.qml

```diff
@@ -15,8 +19,89 @@ Column {
 
     signal goToNetworksView()
 
+
+    StatusBaseText {
+        id: titleText
+        text: qsTr("Wallet")
+        font.weight: Font.Bold
+        font.pixelSize: 28
+        color: Theme.palette.directColor1
+    }
+
+    Item {
+        height: Style.current.bigPadding
+        width: parent.width
+    }
+
+    StatusSettingsLineButton {
+        text: qsTr("Manage Assets & List")
+        height: 64
+        onClicked: goToNetworksView()
+    }
+
+    Separator {
+        anchors.topMargin: Style.current.bigPadding
+        anchors.left: parent.left
+        anchors.leftMargin: -Style.current.padding
+        anchors.right: parent.right
+        anchors.rightMargin: -Style.current.padding
+    }
+
+    StatusSettingsLineButton {
+        text: qsTr("DApp Permission")
+        height: 64
+        onClicked: goToNetworksView()
+    }
+
+    Separator {
+        anchors.topMargin: Style.current.bigPadding
+        anchors.left: parent.left
+        anchors.leftMargin: -Style.current.padding
+        anchors.right: parent.right
+        anchors.rightMargin: -Style.current.padding
+    }
+
     StatusSettingsLineButton {
         text: qsTr("Networks")
+        height: 64
         onClicked: goToNetworksView()
     }
+
+    Separator {
```

Comment:
here giving the separator a height equivalent to the gap in between component would be a better approach ?
from the design it seems that the separator should have a height of 17 ? then we remove the need for the empty item?

Reply:
currently if you notice the highlight as well the overlap the separator and it shouldn't
be that way

Reply:
Good call updated I think